.gitignore

```diff
@@ -78,6 +78,8 @@ locale_stats.json
 
 # node.js
 node_modules/
+package.json
```

Comment:
@HelgiBergmann why did you change this? This might break the ci pipeline as we need the packages for style linting

Reply:
Because I installed ESlint locally with node modules so I do not know should the package.json needs to be in the repo. I will remove than changes.

Reply:
You can install all needed moduls with npm install but the CI should run everything once you are checked for execution